src/components/pages/document.js

```diff
@@ -783,8 +787,14 @@ function Document({ initialState, PageTemplate }) {
     }
   };
 
-  const handleInputViewClick = () => {
-    throw new Error('NOT IMPLEMENTED');
+  const handleViewDocumentInputClick = inputId => {
+    const newSelectedInput = documentInputs.find(x => x._id === inputId);
+    setSelectedDocumentInput(newSelectedInput);
+    const revision = currentDocumentRevisions.find(x => x._id === newSelectedInput.documentRevisionId);
+    setInputsSelectedDocumentRevision(revision);
+    setCurrentSections(revision.sections);
+    setPendingInputValues({});
```

Comment:
I think now we always set this to the default `createEmptyInputsForSections` even when resetting.

Reply:
Changed it so that now inputs are always in sync with the currently displayed sections (setting `{ value: null }` as the default, even if sections do not support input).

Reply:
Much more straight forward and simple, nice!